files-community/Files
# RichCommands: Navigate Up, Back and Forward

**Resolved / Related Issues**
   Related #11481 

**Validation**
How did you test these changes?
   1. Navigate to folders
   2. Use toolbar buttons and shortcuts to navigate up, back and forward
